Prog/vacant-lots-fairfax.Rmd

```diff
@@ -362,65 +330,92 @@ zon <- st_transform(zon, crs = 4326)
 
 ```
 
+
+
+
+  
 Convert vacant lots to SF, set CRS. To convert to SF, we need to use only the observations with non-missing longitude and latitude. We separate these off and recombine after the spatial join.
 
+
+
 ```{r convert-sf}
 
-vacantsf <- vacant1 %>% 
-  filter(missing_coord == 0) %>% 
+jursf <- jur_3 %>% 
+  filter(missing_coord==0) %>% 
   st_as_sf(coords = c("long", "lat")) %>% 
   st_set_crs(st_crs(zon))
 ```
 
-Plot to make sure they are overlapping
-```{r make-map, echo=FALSE}
-# ggplot() +
-#   geom_sf(data = zon, mapping = aes()) +
-#   #scale_fill_manual(values = c("#1696d2", "#fdbf11", 
-#                                #"#d2d2d2", "#ec008b", 
-#                                #"#55b748")) +
-#   geom_sf(data = vacantsf, mapping = aes(),
-#           color = "black")
+
+
+Filter out Herndon and Vienna as these jurisdictions do not fall under Fairfax County zoning.
+```{r}
+options(tigris_use_cache=T)
+places<-places("Virginia") %>% st_as_sf() %>% st_transform(st_crs(zon)) %>% filter(NAME=="Herndon"|NAME=="Vienna") %>% select()
+
+
+jursf_temp<-jursf %>% mutate(notIsPlace= sapply(st_intersects(jursf, places),function(x){length(x)==0})) %>% 
+               filter(notIsPlace)
+
+removed <-  nrow(jursf) - nrow(jursf_temp) 
+
+jursf_temp1 <- jursf_temp %>% mutate(id = seq(1, nrow(jursf_temp), 1))
+
 
 ```
 
-Spatial join
 
+
+Spatial join
 ```{r spatial-join}
 
-vacantjoin <- st_join(vacantsf, zon, join = st_intersects)
+jurjoin <- st_join(jursf_temp1, zon, join = st_intersects)
```

Comment:
jurjoin gains 23 observations over jursf_temp1? I see they get dropped later. how do they get added?

Reply:
This could happen if a point in jursf_temp1 is overlapping more than one zone in zon. I delete the duplicate in these instances.